radlab-ui/webapp/__tests__/utils/variables.test.ts

```diff
@@ -0,0 +1,74 @@
+// Write a unit test for a function that merges multiple objects together.
+
+import { mergeAllSafe } from "@/utils/variables"
+
+describe("mergeAllSafe", () => {
+  it("should two objects together", () => {
+    const adminVars = {
+      organization_id: 123,
+      folder_id: 456,
+      billing_account_id: 789,
+    }
+
+    const moduleVars = {
```

Comment:
Need more info on the logic with which the variables have been associated to `moduleVars` or `userVars` or `expected` have been segregated.

Reply:
These are just a place holder for tests